content/en/docs/hertz/overview/_index.md

```diff
@@ -7,45 +7,50 @@ description: >
 ---
 
 ## CloudWeGo-Hertz
-Hertz [həːts] is a high-usability, high-performance and high-extensibility Golang HTTP framework that helps developers build microservices.
-It was designed with reference to other open-source frameworks like [fasthttp](https://github.com/valyala/fasthttp)、[gin](https://github.com/gin-gonic/gin)、[echo](https://github.com/labstack/echo) and combined with the internal requirements in ByteDance.
-At present, it has been widely used inside ByteDance. Nowadays, more and more microservices use Golang. If you have requirements for microservice performance and hope that the framework can fully meet the internal customizable requirements, Hertz will be a good choice.
+Hertz [həːts] is a high-performance, extensible HTTP framework for Go. It's designed to simplify building microservices for developers.
```

Comment:
We would like to remain the three-high statement while others are well optimized.